sunsynk/sunsynk.py

```diff
@@ -18,6 +18,7 @@ class Sunsynk:
     baudrate: int = attr.ib(default=9600)
     server_id: int = attr.ib(default=1)
     timeout: int = attr.ib(default=10)
+    read_sensors_batch_size: int = attr.field(default=60)
```

Comment:
I opted to make this a field, however it could also be a parameter to the `read_sensors` method if that is preferred.